### ISSUE TITLE
Weekly Scrum - 3/1/2018

### ISSUE THREAD

**Issue by Apocahub** [Apocahub/4806Project#2]

**Comment by Apocahub** [comment]
I have decided upon a project with the group, and setup a public Github building with TravisCI and deploying on Heroku.

I will work on the project with my group, dividing up the work into finite pieces to be developed in parallel.

**Comment by wardandr576** [comment]
I have decided upon a project with the group. A basic model of the application has been developed.
I will be working on assisting with further construction of the model, as well as Junit Testing for each class. Controllers and Repos will need to be set up.
Midterms/4th year project are currently holding me back.

**Comment by keeganwills** [comment]
I last week i did pair programming contributing to the model design of the base of the project. 
Next week i plan on further refining the model, finishing unit tests and the UML diagrams of the code base.

**Comment by wardandr576** [comment]
I have helped implement the model, controllers, and demo.
I will be working on handling additional functionalities for the system in the next week.
Fourth year project is still currently holding me bacl

**Comment by Samadroberts** [comment]
This week I helped decide the goals of the project with the group. I helped coordinate development and implement login functionality on the front and back end. I have also helped clean up and maintain a streamlined code base.

Next week I will be working on implementing unit tests, and implementing further functionalities for professors, and project coordinators.

The obstacles holding me back are limited to completing 4th year project
